src/utils.ts

```diff
@@ -587,7 +585,13 @@ export async function properlyOpenIoTWorkspaceProject(
 }
 
 export function isWorkspaceProject(): boolean {
-  const rootPath = getFirstWorkspaceFolderPath();
+  let rootPath = '';
+  try {
+    rootPath = getFirstWorkspaceFolderPath();
+  } catch (error) {
+    return false;
+  }
+
   const workbenchFileName =
       path.join(rootPath, 'Device', FileNames.iotWorkbenchProjectFileName);
```

Comment:
'Device' could be a constant of FolderNames

Reply:
Fixed.